### PR TITLE
[ENH] Mail course popup prompt upon download

### LINKED ISSUE #98: Mail course popup prompt upon download
We've prepared a short 4-emails-long course with Orange tips and tricks.

A good time to ask a user to subscribe would be after they've decided to try out Orange, and are waiting for it to finish downloading. The popup should fade in a couple of seconds after the user presses the download button, looking something like this:

Screen Shot 2019-10-21 at 15 50 48

, the button linking to https://mailtrain.biolab.si/subscription/JVS_UVh-8 (https certificate to be fixed before merge).

Here's a concept with some starting code https://github.com/irgolic/orange-hugo/commit/22be90562af3b42c2706294603dd7e3e8af6fa22, but I'd rather avoid using an iframe, opting for a button that links to the signup form instead.

### PR DESCRIPTION
**Pull request by mitjapotocin**
Fixes #98

Download page:
* Formatted document
* Fixed Windows tab layout issue
* Added Email course popup

ToDo:
* How do we show the popup to Linux users?
* Subscription link https certificate needs to be fixed before merge.

### CODE REVIEW

File: layouts/download/list.html [hunk 1]
```diff
@@ -1,99 +1,128 @@
 {{ define "main"}}
+<div id="course-popup" class=" popup">
+    <div class="popup-content">
+        <button id="btn-popup-close" class=" btn-popup-close">&times;</button>
+        <img class="popup-img" src="/images/orange_title_scaled.png" alt="">
+        <h2>NEW: Data Science email course [FREE]</h2>
+        <p>45% of Orange users want to learn more about data science (survey, 2017, n=6000). We want to help. Join
+            out email course and get weekly tips.</p>
```
Comment (ajdapretnar) [inline layouts/download/list.html]: out email course - our email course

**Comment by ajdapretnar** [comment]
@irgolic Could you please check subscription link https certificate. The link looks fine to me, but I am probably missing something.

I think we have no other option than not show this for Linux users. Or perhaps we could show them the pop-up after 30s on the website? This is what we had on the old page. Very few Orange users install with Linux, so I suppose we can afford to live with this?

**Comment by irgolic** [comment]
Https should be fixed, I turned on the Cloudflare DNS proxy.

**Comment by ajdapretnar** [comment]
I thought so, it looked fixed to me. Would you say we can merge this and keep Linux site pop-up free?

File: layouts/download/list.html (hunk 1 above)
Comment (irgolic) [reply layouts/download/list.html]: I'm nitpicking, but I'd change the last sentence to 'Join our email course for weekly Orange tips and tricks!'

**Comment by irgolic** [comment]
I think leaving Linux users without a popup is fine. Maybe we should look into showing the course elsewhere too, where Linux users would have a chance to enrol.

**Pull request merged**